### PR TITLE
Reusable sanitary Pads

### PR DESCRIPTION
**Pull request by dawaedina27**

### CODE REVIEW

File: _resources/reusable_sanitary_pads.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+resource:
+    name: 'REUSANBLE SANITARY PADS MAKING'
+    id: 'reusable_sanitary_pads'      #can only contain small letters, numbers, minus and underscore. needs to be the same as the file name
+    description: "Reusable Sanitary Pads are cloth pads that can be used several times. They absorb the menstrual blood and the pad should stay for about 4-5 hours depending on your menstrual flow. After use, you need to wash the cloth really well and remove every stain of blood and then you can reuse the same pad. These cloth pads are usually made in several layers."
+    url: 'https://github.com/dawaedina27/Reusable-Sanitary_Pads_Making#reusanble-sanitary-pads-making'
+    duration: 60     #in minutes
+    difficulty: 2    #1-3, 1 is easyest
+    cost: 0      #in $ (dont leave empty, but type 0 instead)
+    license: #e.g. CC BY-SA 4.0
```
Comment (timmwille) [inline _resources/reusable_sanitary_pads.md:10]:
```suggestion
    license: '[CC-BY-SA 4.0 International](https://choosealicense.com/licenses/cc-by-sa-4.0/)' #e.g. CC BY-SA 4.0
```

File: _modules/Reusable_Sanitary_Pads.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+module:
+    name: 'Reusable Sanitary Pads'
+    id: 'reusable_sanitary_pads'      #can only contain small letters, numbers, minus and underscore. needs to be the same as the file name
+    url: 'https://github.com/dawaedina27/Reusable-Sanitary_Pads_Making'     #url that is linked in the table view, can be empty
+    tag: 'production&supply'    #list, just separated by blank space, e.g. 'Web Open_Source'
+    description: "Reusable Sanitary Pads are cloth pads that can be used several times. They absorb the menstrual blood and the pad should stay for about 4-5 hours depending on your menstrual flow. After use, you need to wash the cloth really well and remove every stain of blood and then you can reuse the same pad. These cloth pads are usually made in several layers."
+    pic-url: 7-3.jpg
+    duration: 48 #put to 0, introduction time in minutes no function currently, the resources have their own time blocks
```
Comment (timmwille) [inline _modules/Reusable_Sanitary_Pads.md:9]:
```suggestion
    duration: 0 #48 but put to 0, introduction time in minutes no function currently, the resources have their own time blocks
```

**Review by timmwille — APPROVED** [review]
approval from my end @walthierer

**Comment by dawaedina27** [comment]
Thanks @timmwille

**Pull request merged**